Lms/expand district nces id size

anathomical opened the pull request:
## WHAT
Increase the size of the `districts.nces_id` from `int` to `bigint`
## WHY
We have a known premium school with a 12-digit NCES ID, and it won't fit in the current column.  This is impacting the Clever integration since it relies on NCES ID to match Clever districts to Quill districts.
## HOW
Just write a simple migration to update the column type

### Notion Card Links
https://www.notion.so/quill/Clever-Sign-in-Issues-Milwaukee-College-Prep-0c0b4f0b99134e48aaec05f889f45163

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  No test coverage on column sizes
Have you deployed to Staging? | Deploying now
Self-Review: Have you done an initial self-review of the code below on Github? | Yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | N/A
Back-to-school 2022: Have you checked the [webinar schedule](https://www.notion.so/quill/Back-to-school-webinar-banners-2022-a75a89cfad9f434899ef6be3eb184733) to avoid for downtime/risky deploys? | N/A

anathomical committed:
Add a migration to make districts.nces_id a bigint instead of int

We have a known school that is paying for premium that has a 12-digit NCES ID, and that ID can't actually be set on the district without this migration.  Having an NCES ID mismatch is disrupting our Clever integration

anathomical committed:
Commit documentation updates from an old migration

anathomical committed:
Fix linting error

happythenewsad commented on services/QuillLMS/db/migrate/20220825144048_incrase_district_nces_id_size.rb:
```diff
@@ -0,0 +1,14 @@
+  def down
+    # NOTE: Once we actually have some columns with bigints in them
+    # this rollback step will start failing until they're removed.
+    # Which is to say, this probably won't actually be easily roll-back-able
```
Good callout. Let's have `down` be a noop, and just leave the comment.

anathomical replied:
@happythenewsad Seems reasonable.  Migration updated.

anathomical committed:
Update migration to no-op when rolled back

Because once data of the new size gets into the database, the rollback won't be possible until/unless it's removed.

anathomical committed:
Merge branch 'develop' into lms/expand-district-nces-id-size
